internal/server/ui/index.html

```diff
@@ -1,62 +1,84 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">

```

Comment:
nitpicking here; Are these necessary?

```suggestion
<html>

```

Reply:
No doctype sometimes causes errors in the browser console, the lang probably isn't but it is good as we only support 1 localization